raiden-ts/src/channels/actions.ts

```diff
@@ -12,15 +12,15 @@ const ChannelId = t.type({
 });
 
 /* A new head in the blockchain is detected by provider */
-export const newBlock = createAction('newBlock', t.type({ blockNumber: t.number }));
+export const newBlock = createAction('new/block', t.type({ blockNumber: t.number }));
```

Comment:
As the actions are like `domain/action`, maybe this should be
```suggestion
export const newBlock = createAction('block/new', t.type({ blockNumber: t.number }));
```

Reply:
That makes absolute sense, let me fix that.